sdk/monitor/azure-monitor-query/azure/monitor/query/_models.py

```diff
@@ -57,14 +57,15 @@ def _from_generated(cls, generated):
         )
 
 
-class LogsTableRow(object):
+class LogsTableRow(list):
```

Comment:
Yes - it's by design that we make this a list. It's basically a custom list where you can access by a column label which is a different list by itself. 
Also, this way, we are able to pass the type directly into pandas, so there's that too.

Reply:
Do you want to update the information in changelog? (and for the new model MetricsList as well)

Reply:
I purposely didn't add it in changelog. There's no behavior difference from a user perspective. 
And for the MetricList, for now, I'm documenting it as a list since it's read only.